### ISSUE TITLE
Network: Firewall: Add Services causes error

### ISSUE THREAD

**Issue by skobyda** [cockpit-project/cockpit#10617]
1) When I try to add the service into firewall, but I choose no services to be added, it causes error:
 
```
Uncaught TypeError: services.map is not a function
    at Object.firewall.addServices (firewall-client.es6:200)
    at clicked (firewall.jsx:268)
    at DialogFooter.action_click (cockpit-components-d…log.jsx?8f9c***:100)
    at HTMLUnknownElement.callCallback (react-dom.development.js?d4d0****:100)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?d4d0****:138)
    at Object.invokeGuardedCallback (react-dom.development.js?d4d0****:187)
    at Object.invokeGuardedCallbackAndCatchFirstError (react-dom.development.js?d4d0****:201)
    at executeDispatch (react-dom.development.js?d4d0****:461)
    at executeDispatchesInOrder (react-dom.development.js?d4d0****:483)
    at executeDispatchesAndRelease (react-dom.development.js?d4d0****:581)
```

**How to replicate:**
1. Go to Networking->Firewall
2. Add Services...
3. Choose no services to be added and save

**Comment by KKoukiou** [comment]
There should be check in the dialog submit handler in order to show error notification if user selected no services.

**Comment by skobyda** [comment]
Might probably send PR myself soon, resolving this issue.

**Comment by martinpitt** [comment]
I fixed this in PR #10642.
